python/src/block/MultiDistributionData.python.cpp

```diff
@@ -26,8 +26,13 @@ void wrapMultiDistributionData( python::module& module, python::module& ) {
 
     module,
     "MultiDistributionData",
-    "Convenience interface for outgoing energy distribution data from the\n"
-    "DLW block for a single reaction using multiple distributions"
+    "The distribution is a combination of multiple distributions\n\n"
+    "The MultiDistributionData class contains a set of distributions and\n"
+    "associated probabilities to represent the particle's distribution data.\n"
+    "This type of distribution is rarely used, and when it happens it is often\n"
+    "used for higher chance fission spectra. It is used in the DLW block (there\n"
+    "is no ACE LAW associated to this distribution type, but internally in ACEtk\n"
+    "we refer to it as ACE LAW 666 because it is evil)."
```

Comment:
Every distribution needs an associated enum value for the distribution type. SInce the multi-distribution does not correspond to an ACE LAW I assigned it as 666. I really don't like the way ACE does the multi-distribution hence the number.

I'm happy you noticed - it means you're paying attention ;-)